Assets/Mirror/Runtime/NetworkManagerHUD.cs

```diff
@@ -85,13 +85,7 @@ void OnGUI()
                 // server / client status message
                 if (NetworkServer.active)
                 {
-                    string serverMsg = "Server: port=" + manager.networkPort;
-                    if (manager.useWebSockets)
-                    {
-                        serverMsg += " (Using WebSockets)";
-                    }
-
-                    GUILayout.Label(serverMsg);
+                    GUILayout.Label("Server: port=" + manager.networkPort);
```

Comment:
Too limited.    I would like to know which transport this is, and potentially other properties.
call manager.transport.ToString()  instead,  and let the transport decide what to display here.